bundles/org.openhab.binding.upnpcontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -23,52 +36,163 @@
 			<channel id="tracknumber" typeId="tracknumber"/>
 			<channel id="trackduration" typeId="trackduration"/>
 			<channel id="trackposition" typeId="trackposition"/>
+			<channel id="reltrackposition" typeId="reltrackposition"/>
 		</channels>
+		<representation-property>udn</representation-property>
 		<config-description>
 			<parameter name="udn" type="text" required="true">
 				<label>Unique Device Name</label>
 				<description>The UDN identifies the UPnP Renderer</description>
 			</parameter>
+			<parameter name="refresh" type="integer" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds</description>
+				<default>60</default>
+			</parameter>
+			<parameter name="notificationVolumeAdjustment" type="integer" min="-100" max="100" step="1" unit="%">
+				<label>Notification Sound Volume Adjustment</label>
+				<description>Specifies the percentage adjustment to the current sound volume when playing notifications</description>
+				<default>10</default>
+			</parameter>
+			<parameter name="maxNotificationDuration" type="integer" unit="s">
+				<label>Maximum Notification Duration</label>
+				<description>Specifies the maximum duration for notifications, longer notification sounds will be interrupted. O
+					represents no maximum duration</description>
+				<default>15</default>
+			</parameter>
+			<parameter name="seekStep" type="integer" min="1">
+				<label>Fast Forward/Rewind Step</label>
+				<description>Step in seconds for fast forward rewind</description>
+				<default>5</default>
+			</parameter>
+			<parameter name="responseTimeout" type="integer" unit="ms">
+				<label>UPnP Response Timeout</label>
+				<description>Specifies the timeout in milliseconds when waiting for responses on UPnP actions</description>
+				<default>2500</default>
+				<advanced>true</advanced>
+			</parameter>
 		</config-description>
-
 	</thing-type>
+
 	<thing-type id="upnpserver">
 		<label>UPnPServer</label>
 		<description>UPnP AV Server</description>
 		<channels>
 			<channel id="upnprenderer" typeId="upnprenderer"/>
-			<channel id="currentid" typeId="currentid"/>
+			<channel id="currenttitle" typeId="system.media-title"/>
```

Comment:
Is this breaking backward compatibility?

Reply:
@fwolter Yes, it is. The browse channel now is a state channel and contains the info. The browse channel used to be a command channel. I could keep the channel, but I am not sure it is worth it. Will the change from command to state channel for browse not be breaking as well? And having it on the browse channel is important, I noticed, when setting up a UI in OH3.

Reply:
Just asking. The current users need to remove and re-add the Thing to make it work again after an update. If you confirm, I'll add the "(potentially) not backward compatible" label.

Reply:
Yes, I believe it is best to remove the channel. So, it will not be backward compatible.